cmd/experimental/kjobctl/pkg/cmd/create/create.go

```diff
@@ -328,10 +461,67 @@ func (o *CreateOptions) Complete(clientGetter util.ClientGetter, cmd *cobra.Comm
 		}
 	}
 
-	if cmd.Flags().Changed(podRunningTimeout) && o.PodRunningTimeout <= 0 {
+	if cmd.Flags().Changed(podRunningTimeoutFlagName) && o.PodRunningTimeout <= 0 {
 		return errors.New("--pod-running-timeout must be higher than zero")
 	}
 
+	if (o.SlurmFlagSet.Changed(memPerTaskFlagName) && (o.SlurmFlagSet.Changed(memPerCPUFlagName) || o.SlurmFlagSet.Changed(memPerGPUFlagName))) ||
+		o.SlurmFlagSet.Changed(memPerTaskFlagName) || (o.SlurmFlagSet.Changed(memPerCPUFlagName) && o.SlurmFlagSet.Changed(memPerGPUFlagName)) {
+		return errors.New("the --mem-per-task, --mem-per-cpu and --mem-per-gpu options are mutually exclusive")
+	}
```

Comment:
Is it possible to use cobra MarkFlagsMutuallyExclusive?

Reply:
Changed using a custom function